src/main/java/com/hunllefhelper/HunllefHelperConfig.java

```diff
@@ -1,5 +1,7 @@
 package com.hunllefhelper;
```

Comment:
It's nice to have the config files in a seperate folder. Could you move this file to the config package as well?